test-harness/handlers/testerror.cfc

```diff
@@ -0,0 +1,9 @@
+component output="false"{
```

Comment:
testing coldbox error template and Visually we can analise error info. 

At the moment in CF11 the error template does not display error-detail + error-message